spatial/r3/mat_test.go

```diff
@@ -224,9 +223,8 @@ func TestRotationFromQuat(t *testing.T) {
 
 func BenchmarkQuat(b *testing.B) {
 	rnd := rand.New(rand.NewSource(1))
-	m := NewMat(nil)
 	for i := 0; i < b.N; i++ {
 		q := quat.Number{Real: rnd.Float64(), Imag: rnd.Float64(), Jmag: rnd.Float64(), Kmag: rnd.Float64()}
-		m.RotationFromQuat(q)
+		Rotation(q).Mat()
```

Comment:
Should this assign to a global variable?

Reply:
I'll do a nil test on it.